decam/camGeom/camera.py

```diff
@@ -1,5 +1,6 @@
 import lsst.afw.cameraGeom.cameraConfig
-assert type(config) == lsst.afw.cameraGeom.cameraConfig.CameraConfig, 'config is of type %s.%s instead of lsst.afw.cameraGeom.cameraConfig.CameraConfig' % (
+assert type(config) == lsst.afw.cameraGeom.cameraConfig.CameraConfig, \
+    'config is of type %s.%s instead of lsst.afw.cameraGeom.cameraConfig.CameraConfig' % (
     type(config).__module__, type(config).__name__)
```

Comment:
This doesn't feel like a `black`/`ruff` style reformatting? I don't think it reads as cleanly either. Wouldn't something more like this read better?
```
assert (
    type(config) == lsst.afw.cameraGeom.cameraConfig.CameraConfig
), "config is of type %s.%s instead of lsst.afw.cameraGeom.cameraConfig.CameraConfig" % (
    type(config).__module__,
    type(config).__name__,
)
```

Reply:
This is something that we write during config serialization and not something that we usually include in configs that we commit to obs packages. Maybe it's special because it's the camera geometry?

I should probably change pex_config to write it like this:

```python
assert type(config) == lsst.afw.cameraGeom.cameraConfig.CameraConfig, (
    f"config is of type {type(config).__module__}.{type(config).__name__} instead of lsst.afw.cameraGeom.cameraConfig.CameraConfig"
)
```

Reply:
Using f-strings, my formatter recommends this:
```python
assert (
    type(config) == lsst.afw.cameraGeom.cameraConfig.CameraConfig
), f"config is of type {type(config).__module__}.{type(config).__name__} instead of lsst.afw.cameraGeom.cameraConfig.CameraConfig"
```

Also, this:
```
do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`Flake8(E721)
```

Reply:
Yes. I ran black, then ruff to fix up the f-string, and then forgot to run black again...

The `==` rather than `is` has been there since 2013...

Reply:
So, ultimately then, something like this would be optimal?
```python
assert type(config) is lsst.afw.cameraGeom.cameraConfig.CameraConfig, (
    f"config is of type {type(config).__module__}.{type(config).__name__} "
    "instead of lsst.afw.cameraGeom.cameraConfig.CameraConfig"
)
```

Reply:
`black` didn't reformat this line for me, I did it myself. Your last option is fine by me.